### ISSUE TITLE
Setup NuxtJS

### ISSUE THREAD

**Issue by mmuller88** [monadahq/website#1]
Story:
As a cloud developer, I want to see more information on monada.co 

Acceptance Criteria:
- [ ] NuxtJS was setup and it uses the current HTML content from monada.co

Info:
* Quick Start: https://v3.nuxtjs.org/getting-started/quick-start
* And then, the Guide: https://v3.nuxtjs.org/guide/concepts/introduction

**Comment by mmuller88** [comment]
Story:
As a cloud developer, I want to see more information on monada.co 

Acceptance Criteria:
- [ ] NuxtJS was setup and it uses the current HTML content from monada.co

Info:
* helping material

**Comment by noyfactor** [comment]
Wait what? Why Nuxt?

**Comment by skyrpex** [comment]
Martin wants to do some frontend work and that's what I recommended to him, but we all need to discuss what to use before starting working on it, for sure!

We'll bring the subject during the daily

**Comment by mmuller88** [comment]
This is on hold for now. We would like to clarify ownership first.
